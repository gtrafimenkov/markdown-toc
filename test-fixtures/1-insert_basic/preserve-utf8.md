# Title标题
# Überschrift