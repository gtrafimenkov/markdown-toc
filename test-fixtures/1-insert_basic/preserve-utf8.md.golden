- [Title标题](#title标题)
- [Überschrift](#überschrift)

# Title标题
# Überschrift